core/src/OpenMPTarget/Kokkos_OpenMPTarget_Exec.hpp

```diff
@@ -410,6 +410,10 @@ class TeamPolicyInternal<Kokkos::Experimental::OpenMPTarget, Properties...>
            team_size_ * m_thread_scratch_size[level];
   }
 
+  inline Kokkos::Experimental::OpenMPTarget space() const {
```

Comment:
Why do you say it's superfluous here? AFAICT `inline` here is still meaningful to many compilers. It tells the compiler to contribute to the `inline` function inlining budget rather than the regular inlining budget (which is typically smaller). See, for instance, the GCC optimization parameter `max-inline-insns-auto` (at https://gcc.gnu.org/onlinedocs/gcc/Optimize-Options.html)

> When you use -finline-functions (included in -O3), a lot of functions that would otherwise not be considered for inlining by the compiler will be investigated. To those functions, a different (more restrictive) limit compared to functions declared inline can be applied. The default value is 100.

Reply:
Declared in the body of the class https://isocpp.org/wiki/faq/inline-functions#inline-member-fns-more